components/core/main.conf

```diff
@@ -0,0 +1,3 @@
+{"user": {"username": "", "password": ""},
```

Comment:
Lets move user credentials out from the config files to the environment variables